src/world.h

```diff
@@ -1,6 +1,8 @@
 #ifndef _world_h_
 #define _world_h_
 
+// #include <time.h>
```

Comment:
I think you can take out extraneous stuff like this.

Reply:
Yeah, I'll take this out.